lib/helm_deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+NAMESPACE=$1
+CHART_PATH=$2
+TAG=$3
+echo "Creating Helm installation in $NAMESPACE namespace with tag $TAG"
+
+helm upgrade --install --atomic --set image.app.tag=$TAG -n $NAMESPACE connectivity-intake $CHART_PATH --debug
```

Comment:
I'd lean towards explicitly passing the default values as well, which will make it clear that [our manually set keys are going to persistently override any default values](https://helm.sh/docs/chart_template_guide/values_files/) and make it easy to chain additional environment-specific defaults as needed moving forward.
```suggestion
helm upgrade --install --atomic --values "./values.yml" --set image.app.tag=$TAG -n $NAMESPACE connectivity-intake $CHART_PATH --debug
```